bindings/dart/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -363,6 +363,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
was this automatically added by xcode?
because normally that is only set for debug but not for profile or release build.

Reply:
i think this was added by my xcode when i tried to make it work that xcode builds&runs the flutter app (via the gui). but since this still doesn't work and we have to call `flutter run` anyways, i'll try to revert this.